src/Frontend/src/views/throughputreport/setup/ConnectionSetupView.vue

```diff
@@ -0,0 +1,118 @@
+          {{ transportNameForInstructions() }}.
+        </p>
+        <ConfigurationCode :settings="settingsInfo?.broker_settings ?? []">
+          <template #configInstructions>
+            <div>Paste the settings above in the <code>ServiceControl.exe.config</code> file of the ServiceControl Error instance.</div>
+          </template>
+          <template #environmentVariableInstructions>
+            <div>Execute the above instructions in a terminal to set the environment variables, these variables need to be set for the user that in running the ServiceControl Error instance.</div>
+          </template>
+        </ConfigurationCode>
+      </div>
+    </div>
+  </template>
+  <template v-if="useIsMonitoringEnabled()">
+    <div class="row configuration">
+      <div class="col-12">
+        <h4>ServiceControl Settings</h4>
+        <p class="nogap">
+          Settings to ensure that usage data is being collected from the Monitoring instance.<br />
+          For more information read the <a href="https://docs.particular.net/servicecontrol/creating-config-file">ServiceControl</a> settings documentation.
+        </p>
+        <ConfigurationCode :settings="settingsInfo?.service_control_settings ?? []">
+          <template #configInstructions>
+            <div>Paste the settings above in the <code>ServiceControl.exe.config</code> file of the ServiceControl Error instance.</div>
+          </template>
+          <template #environmentVariableInstructions>
+            <div>Execute the above instructions in a terminal to set the environment variables, these variables need to be set for the user that in running the ServiceControl Error instance.</div>
+          </template>
+        </ConfigurationCode>
+      </div>
+    </div>
+    <div class="row configuration">
+      <div class="col-12">
+        <h4>Monitoring Settings</h4>
+        <p class="nogap">
+          Settings to ensure that usage data is being collected from the Monitoring instance.<br />
+          These settings do not need to be modified unless MSMQ transport is used with the Monitoring instance installed on a different machine to the ServiceControl Error instance.<br />
+          For more information read the <a href="https://docs.particular.net/servicecontrol/monitoring-instances/installation/creating-config-file">Monitoring</a> settings documentation.
+        </p>
+        <ConfigurationCode :settings="settingsInfo?.monitoring_settings ?? []" configFileName="ServiceControl.Monitoring.exe.config">
+          <template #configInstructions>
+            <div>Paste the settings above in the <code>ServiceControl.Monitoring.exe.config</code> file of the ServiceControl Monitoring instance.</div>
+          </template>
+          <template #environmentVariableInstructions>
+            <div>Execute the above instructions in a terminal to set the environment variables, these variables need to be set for the user that in running the ServiceControl Monitoring instance.</div>
```

Comment:
### 
```suggestion
            <div>Execute the above instructions in a terminal to set the environment variables -, these variables need to be set for the account under which the ServiceControl Monitoring instance is running.</div>
```

Reply:
it seems you have an extra `-`, is this on purpose?

Reply:
it was not on purpose